src/interface/sparse_pack_base.cpp

```diff
@@ -148,18 +156,8 @@ SparsePackBase SparsePackBase::Build(T *pmd, const PackDescriptor &desc) {
                     PARTHENON_REQUIRE(pack_h(1, b, idx).size() ==
                                           pack_h(0, b, idx).size(),
```

Comment:
I think this check can probably be removed as well. SparsePacks are probably not where we need to be ensuring that the fluxes have the correct size and we may move to the fluxes having a different size (to conform to the sizes of face variables for instance).

Reply:
Done